starport/services/network/address.go

```diff
@@ -0,0 +1,14 @@
+package network
```

Comment:
Thanks for implementing it, it's needed for prepare as well, so we can create a specific PR for it

Reply:
https://github.com/tendermint/starport/pull/1801